bin/_mb.bat

```diff
@@ -29,7 +29,7 @@ if "%PYBLISH_NUKE%"=="" goto :missing
 if "%PYBLISH_LITE%"=="" goto :missing
 if "%PYBLISH_QML%"=="" goto :missing
 if "%MINDBENDER_CORE%"=="" goto :missing
-:: if "%PYBLISH_USERNAME%"=="" goto :missing
+if "%PYBLISH_USERNAME%"=="" goto :missing
```

Comment:
And what should the value be?

1. If we set it in the local `mb.bat`, then the value would be the same for everyone who uses it, which doesn't seem right. 
2. If the value is `%USERNAME%` then it might as well be used directly.